module/main.go

```diff
@@ -12,11 +12,30 @@ import (
 	viamcartographer "github.com/viamrobotics/viam-cartographer"
 )
 
+// Versioning variables which are replaced by LD flags.
+var (
+	Version     = "development"
+	GitRevision = ""
+)
+
+// Arguments for the command.
+type Arguments struct {
+	Version bool `flag:"version,usage=print version"`
```

Comment:
[note to self] this is using a go struct tag (https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go).

in particular, the "flag" tag is indicating that if `--version` is passed on the command line, then the "Version" bool should be set to true. the "usage" tag is indicating that if `--help` (or equivalent) is passed on the command line, then the description for Version should be "print version"